flink-contrib/flink-storm-compatibility/flink-storm-compatibility-examples/src/main/java/org/apache/flink/stormcompatibility/wordcount/SpoutSourceWordCount.java

```diff
@@ -145,7 +145,7 @@ private static boolean parseParameters(final String[] args) {
 		}
 
 		return env.addSource(new StormFiniteSpoutWrapper<String>(new StormInMemorySpout(WordCountData.WORDS), true),
-				TypeExtractor.getForClass(String.class));
+				TypeExtractor.getForClass(String.class)).setParallelism(1);
```

Comment:
I thought it was a mistake, because without setting the parallelism to 1, the input will be duplicated. It does not affect the ITCases directly, though, because they run in 'file input/output' mode, but it still seems out of order.
Any reason against setting the parallelism?